### ISSUE TITLE
Implement programming Interface to Stacie-D

### ISSUE THREAD

**Issue by RobertK66** [RobertK66/obc_1769_core#27]
If I understand it correct, the chapter "3.2.4 PRG Connectors" in the new Document "STACIE Description V042" it will be possible to implement a 'Program via UART'-Proxy feature to be used for Flashing Stacie-D Subsystems.

It has to be defined how the exact protocol and timimng together with the mode GPIO should be handeled. At least the OBC could 'proxy' the uart to simulate the PRG Connector from the UMBILICAL-debug_uart after the cubesat is assembeled and no more access to PRG connectors is possible.

**Comment by RobertK66** [comment]
@WolfgangTreb , @AndyFHWN: not sure which document - from last summer - i refer here ;-) ...
Do you have any info/news thoughts on that one!?

**Comment by AndyFHWN** [comment]
Stacie-D will be the exact same module, no changes to existing document here. Do you have it? 
Edit:
Right now i only find "STACIE-D Description V03" on our drive, i will check for stated document

**Comment by RobertK66** [comment]
No, that's not true ;-). There are agreed changes (in hardware) and a lot of proposals how to tackle Software errors done and found in Pegasus but no common agreed decisions. 

And no, I do not have a valid set of Pegasus Documents any more. And there are (to my knowledge) no CLIMB version of this interface documents yet.

I do have your mentioned  "STACIE-D Description V03" on paper. But I am not sure if my issue here referenced a 'real version' or my 'V042' was me - to lazy to look up the real version 😄 ...

Nevertheless this issue could be an enhancement to be able to flash the Stacie-Delta firmware. Not agreed yet if this is feasible at all. (Therefore i tagged it as 'future use'.and it has low priority, other than #69)
